src/generators/number/index.ts

```diff
@@ -1,16 +1,25 @@
-import {Chance} from 'chance';
+import { Chance } from 'chance';
 
 /**
- * Generates a random `number`, optionally between `lowerInclusive` and `upperExclusive`.
+ * Generates a random `number`, optionally between `lowerInclusive` and `upperExclusive` with `fixed` number of digits after the decimal.
  */
 const createNumberGenerator = (chance: Chance.Chance) => (
   lowerInclusive = -1000000,
   upperExclusive = 1000000,
-): number =>
-  chance.floating({
+  fixed = 4,
+): number => {
+  const EPSILON = 0.0000001;
+  const MAX_PRECISION = 6;
+
+  if (upperExclusive != null && upperExclusive > MAX_PRECISION) {
+    throw new TypeError(`"upperExclusive" must be less than or equal to ${MAX_PRECISION} or null`);
+  }
```

Comment:
add a test for this new behavior? (please create a `test.ts` right next to this file and remove [this](https://github.com/goodeggs/fake-eggs/blob/14b431ae025e5aabd49276c5e645ae547e060acb/src/index.test.ts#L25))

Reply:
Addressed in 910d483